src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -108,6 +109,19 @@ public void testCollectNonNumericMetric() {
         assertEquals(expectedLabels, snapshot.getDataPoints().get(0).getLabels());
     }
 
+    @Test
+    public void testLabelNameSanitizing() {
+        String labelName = PrometheusNaming.sanitizeLabelName("k-1");
+        assertEquals("k_1", labelName);
+    }
+
+    @Test
+    public void testLabelDuplicateWarning() {
+        Labels labels = Labels.builder().label("k1", "v1").build();
+        InfoSnapshot.InfoDataPointSnapshot snapshot = DataPointSnapshotBuilder.infoDataPoint(labels, "v1", "k1");
```

Comment:
We should use a different value than `v1` here. Otherwise this does not assert we did not replace the value.

Reply:
Yes, I missed that. Changed to just `value`.